File: iOS/meApp/Domain/Models/Domain/Auth/Integrations.swift
import Foundation

struct Integrations: Codable, Equatable {
    let isFitbitOn: Bool
    let isGoogleFitOn: Bool
    let isMFPOn: Bool
    let isUAOn: Bool
    let isFitbitValid: Bool
    let isGoogleFitValid: Bool
    let isMFPValid: Bool
    let isUAValid: Bool
    let healthkit: Bool
    let isHealthConnectOn: Bool
}

File: Android/app/src/main/java/com/dmdbrands/gurus/weight/features/integration/model/Integrations.kt
package com.dmdbrands.gurus.weight.features.integration.model

// 1. Your domain model as a Kotlin data class
data class Integrations(
  val isFitbitOn: Boolean = false,
  val isGoogleFitOn: Boolean = false,
  val isMFPOn: Boolean = false,
  val isUAOn: Boolean = false,
  val isFitbitValid: Boolean = false,
  val isGoogleFitValid: Boolean = false,
  val isMFPValid: Boolean = false,
  val isUAValid: Boolean = false,
  val healthkit: Boolean = false,
  val isHealthConnectOn: Boolean = false,
)

